common/src/main/java/io/homeassistant/companion/android/common/sensors/LastUpdateManager.kt

```diff
@@ -76,9 +76,12 @@ class LastUpdateManager : SensorManager {
         } else {
             sensorDao.add(SensorSetting(lastUpdate.id, SETTING_ADD_NEW_INTENT, "false", SensorSettingType.TOGGLE))
         }
-        for (setting in allSettings) {
-            if (setting.value == "")
-                sensorDao.removeSetting(lastUpdate.id, setting.name)
+
+        val settingsToRemove = allSettings
+            .filter { setting -> setting.value == "" }
+            .map { setting -> setting.name }
+        if (settingsToRemove.isNotEmpty()) {
+            sensorDao.removeSettings(lastUpdate.id, settingsToRemove)
```

Comment:
This change will cause a conflict with my open PR: #2585 _Fix removing non-last list items from Last Update Trigger sensor's settings_

See [these lines](https://github.com/home-assistant/android/blob/f8f32951447caef3de1ce739e976313365f12d2e/common/src/main/java/io/homeassistant/companion/android/common/sensors/LastUpdateManager.kt#L68-L74) in my changes

We should decide on who and how to resolve it.

Reply:
This change will cause a conflict with my open PR: #2585 _Fix removing non-last list items from Last Update Trigger sensor's settings_

See [these lines](https://github.com/home-assistant/android/blob/f8f32951447caef3de1ce739e976313365f12d2e/common/src/main/java/io/homeassistant/companion/android/common/sensors/LastUpdateManager.kt#L68-L74) in my changes.

We should decide on who and how to resolve it.
I think it would be easier to solve the conflict in your PR because I have several commits which would be harder to rebase. And my PR is already reviewed (but not yet formally approved) by one person (@jpelgrom).